src/settings/Settings.tsx

```diff
@@ -394,12 +387,6 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         default: true,
         controller: new UIFeatureController(UIFeature.Widgets, false),
     },
-    "MessageComposerInput.showLocationButton": {
-        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
-        displayName: _td('Enable location sharing'),
-        default: true,
-        controller: new IncompatibleController("feature_location_share", false, false),
-    },
```

Comment:
Any reason this setting is being removed? Stickers have a similar setting

Reply:
More settings === bad?  The guide from Product was to remove it completely.  It was originally added to help privacy-conscious users, but there was push-back from the community that it was pointless.

Reply:
Sure, as long as it was a request by product then all good! Now with most of the composer actions living in an overflow menu it is much less of a concern for screen real estate anyway